AboutMe.html

```diff
@@ -0,0 +1,29 @@
+<html>
+<head>
+<title> About Me</title>
+<meta charset="utf-8">
+<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+        <header>
+    <nav class="bar">
+          <a  href="AboutMe.html"> About Me </a> |
+          <a class="active" href="index.html"> <i>Portfolio</i> </a>
+      </nav> 
+      <h1 id="only"> Joe Johnson</h1>
+    </header>
+    <h2 id="ENJOY">Experience</h2> <br> 
```

Comment:
be careful with naming element IDs, stay consistent so its easier to remember and you have less future problems